stumpy/scrump.py

```diff
@@ -415,10 +438,6 @@ def _prescrump(
     indices : numpy.ndarray
         The subsequence indices to compute `prescrump` for
 
-    idx_ranges : numpy.ndarray
-        The (inclusive) start indices and (exclusive) stop indices referenced
-        in the `indices` array
```

Comment:
[irrelevant to this PR]
I tried to modify the docstring.py so that it can capture a case like this, where docstring han the description of a variable that is not in the signature. we can start by narrowing down the discovered variables in a docstring to the ones that are before Return. I tried to consider return with/without s and colon in regex. but I wasn't successful since the term return might appear more than once in a docstring. For example, a sentence in docstring might contain this word.

Maybe the following solution works:
We may try to search for different versions of r"Return\n---------" and find its position in the docstring, then read the docstring upto that position, and then use python set method symmetric_difference() to capture any variable that is in the signature or docstring but not both.

Reply:
I don't think I understand why you are trying to do this. Please create a new issue if this is irrelevant to this PR